src/server/services/brp.ts

```diff
@@ -80,6 +66,11 @@ export function transformBRPNotifications(data: BRPData, compareDate: Date) {
     }
   );
 
+  const documentsExpiringDuringPeak = data.identiteitsbewijzen?.filter(
+    (document) => {
+      return new Date(document.datumAfloop) < new Date('2024-10-01') 
+    }
+  );
```

Comment:
Kun je hier niet beter ook de start datum van de melding neerzetten?

```suggestion
  // Vanaf wanneer de melding laten zien
  const DATE_PEAK_PERIOD_NOTIFICATION_START = new Date('2024-02-01');
  // Einde van de piekperiode
  const DATE_PEAK_PERIOD_END = new Date('2024-10-01');
  // Documenten die voor het einde van de piekperiode aflopen
  const documentsExpiringDuringPeak = data.identiteitsbewijzen?.filter(
    (document) => {
      const datumAfloop = new Date(document.datumAfloop);
      return datumAfloop >= DATE_PEAK_PERIOD_NOTIFICATION_START && datumAfloop <= DATE_PEAK_PERIOD_END 
    }
  );
```